### PR TITLE
Texture changes and new logos

### PR DESCRIPTION
**Pull request by Coach-mdl**
Adjusted shading on the bladders and leather sac. Don't worry, no shape changes. Iron flasks have been given water levels like on the old 1.12 textures. I included a 3D iron waterflask in models if you want to experiment with a 3D model when you (V) place it but there's no obligation. It's a placeholder and can be changed. I don't know if it will even work but it would look nice in Firmalife jarbnets. I've included two logos. One is _something_ and the other looks more like the original. Your choice, I just included both to cover bases.

### CODE REVIEW

**Review by Gaelmare — CHANGES_REQUESTED** [review]
Texture paths for 3D model don't seem right?

**Comment by Coach-mdl** [comment]
Where should I leave it?

**Comment by Coach-mdl** [comment]
I went and removed the 3D model for now. Not needed right now.

File: src/main/resources/assets/waterflasks/textures/item/leather_flask.png
Comment (Gaelmare) [inline src/main/resources/assets/waterflasks/textures/item/leather_flask.png:1]: This isn't right, the full flask looks empty now.

Comment (Gaelmare) [reply src/main/resources/assets/waterflasks/textures/item/leather_flask.png:1]: I've moved the flask textures around to make sense, but if you had time to contribute a different leather_flask-1.png that would show a difference between 20% and 40% full, that would be great!

Comment (Coach-mdl) [reply src/main/resources/assets/waterflasks/textures/item/leather_flask.png:1]: Whoops. Misclicked. Don't worry I am about to do it

**Pull request merged**